src/options/options.css

```diff
@@ -204,6 +204,7 @@ fieldset {
 }
 #heatmap {
   margin-left: -10px;
+  /* font-size: 12px; */
```

Comment:
Did you mean for this to be uncommented? Having this commented line does not change the style. Could you update this file and uncomment this line?